acmw-db-app/pages/api/meeting/[pid].js

```diff
@@ -50,6 +50,46 @@ async function meetingAttendance(meetingId) {
     return data.rows;
 }
 
+// POST /api/meeting/create
+// Create a meeting: meeting_name, meeting_date, semester (AUXX or SPXX), and company_name associated (null if no company)
+async function create(meeting_name, meeting_date, semester, company_name) {
```

Comment:
I'd prefer this to take the company id instead of company name since I'm imagining for the front-end piece, we can use a drop down and/or search for companies and send the selected id stored in the option value to this end point. Idk using strings just weirds me out bc you have to get an exact match lol